src/views/Squad.vue

```diff
@@ -1,7 +1,10 @@
 <template>
   <div class="page-wrapper">
     <div class="title">
-      <h1 v-if="currentSquad" class="squad-name">Squad: {{ currentSquad.game }}</h1>
+      <h1 v-if="currentSquad" class="squad-name">
+        Squad: {{ currentSquad.game }}
+      </h1>
+      <p v-if="user === null" class="infos-label">Please login to join</p>
```

Comment:
i think we should remove this line since we now allow joining squads without login

Reply:
I think it was an artifact from when i did squad join earlier when login was required